lib/src/migrators/module.dart

```diff
@@ -184,13 +185,55 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
 
     if (namespace == null) {
       if (!builtInFunctionModules.containsKey(name)) return;
+
       namespace = builtInFunctionModules[name];
       name = builtInFunctionNameChanges[name] ?? name;
+      if (namespace == 'color' && removedColorFunctions.containsKey(name)) {
+        if (node.arguments.positional.length == 2 &&
+            node.arguments.named.isEmpty) {
+          _patchRemovedColorFunction(name, node.arguments.positional.last);
+          name = 'adjust';
+        } else if (node.arguments.named.containsKey('amount')) {
+          var arg = node.arguments.named['amount'];
+          _patchRemovedColorFunction(name, arg,
+              existingArgName: _findArgNameSpan(arg));
+          name = 'adjust';
+        } else {
+          print(node.arguments.named.keys);
+          warn("Could not migrate malformed '$name' call", node.span);
+        }
+      }
       _additionalUseRules.add("sass:$namespace");
     }
     if (namespace != null) patcher(name, namespace);
   }
 
+  /// Given a named argument [arg], returns a span from the start of the name
+  /// to the start of the argument itself (e.g. "$amount: ").
+  FileSpan _findArgNameSpan(Expression arg) {
+    var start = arg.span.start.offset - 1;
+    while (arg.span.file.getText(start, start + 1) != r'$') {
+      start--;
+    }
+    return arg.span.file.span(start, arg.span.start.offset);
+  }
+
+  /// Patches the amount argument [arg] for a removed color function
+  /// (e.g. `lighten`) to add the appropriate name (such as `$lightness`) and
+  /// negate the argument if necessary.
+  void _patchRemovedColorFunction(String name, Expression arg,
+      {FileSpan existingArgName}) {
+    var parameter = removedColorFunctions[name];
+    var leftParen =
+        parameter.endsWith('-') && arg is BinaryOperationExpression ? '(' : '';
+    if (existingArgName == null) {
+      addPatch(patchBefore(arg, '$parameter$leftParen'));
+    } else {
+      addPatch(Patch(existingArgName, '$parameter$leftParen'));
+    }
+    if (leftParen == '(') addPatch(patchAfter(arg, ')'));
```

Comment:
Nit: rather than checking `leftParen` here, I'd just do:

```dart
var needsParens =
    parameter.endsWith('-') && arg is BinaryOperationExpression;
var leftParen = needsParens ? '(' : '';
```

and check `needsParens` here.